src/app/pages/office/cat-manoeuvre/office.cat-manoeuvre.module.ts

```diff
@@ -16,23 +16,25 @@ import {
 
```

Comment:
This will all be failing linting so can we please correct

Reply:
its been fixed a while back